src/selectors/entities/teams.js

```diff
@@ -139,14 +140,23 @@ export const getJoinableTeamIds = createIdsSelector(
 export const getJoinableTeams = createSelector(
     getTeams,
     getJoinableTeamIds,
-    (teams, joinableTeamIds) => {
+    getCurrentUser,
+    (teams, joinableTeamIds, currentUser) => {
         const openTeams = {};
 
         for (const id of joinableTeamIds) {
             openTeams[id] = teams[id];
         }
 
-        return openTeams;
+        function sortTeams(a, b) {
+            const options = {
+                numeric: true,
+                sensitivity: 'base',
+            };
+            return a.display_name.localeCompare(b.display_name, currentUser.locale || 'en', options);
+        }
+
+        return Object.values(openTeams).sort(sortTeams);
```

Comment:
Is this selector being used anywhere already? Wondering if it's better to create a new selector instead of breaking the existing one by modifying it

Reply:
Yes, at webapp (like `TeamStore.getTeamListings()`) and mobile.
Okay, I'll just create `getSortedJoinableTeams `